tests/migrations/test_writer.py

```diff
@@ -7,6 +7,7 @@
 import re
 import tokenize
 import unittest
+import functools
```

Comment:
You'll need to order your imports to appease isort here; `functools` comes before `math`.